src/puppetlabs/puppetdb/pql/transform.clj

```diff
@@ -2,15 +2,21 @@
   (:require [clojure.string :as str]
             [puppetlabs.puppetdb.cheshire :as json]))
 
+(defn paging-clause?
```

Comment:
Is there some way to get this from instaparse, as these are all part of the <pagingclause> rule? Wouldn't be surprised if we could put this as metadata on the AST.